### PR TITLE
Erlebnisgeologie

### PR DESCRIPTION
**Pull request by rebert**

### CODE REVIEW

**Comment by Simon1193** [comment]
hello @rebert 
Could you provide a testlink for this one?

**Comment by rebert** [comment]
@Simon1193 Sorry, It's a pleasure:
https://mf-chsdi3.dev.bgdi.ch/ltrea_erlebnis_geologie/shorten/890a508489

BR

**Comment by Simon1193** [comment]
> @Simon1193 Sorry, It's a pleasure:
> https://mf-chsdi3.dev.bgdi.ch/ltrea_erlebnis_geologie/shorten/890a508489
> 
> BR

thanks!

File: chsdi/models/vector/stopo.py [hunk 1]
```diff
@@ -349,18 +349,57 @@ class GravimetrischerAtlasMesspunkte(Base, Vector):
 register('ch.swisstopo.geologie-gravimetrischer_atlas.messpunkte', GravimetrischerAtlasMesspunkte)
 
 
+class GeologieGeoevents:
+    __table_args__ = ({'schema': 'geol', 'autoload': False})
+    __template__ = 'templates/htmlpopup/geologie_geoevents.mako'
+    __bodId__ = 'ch.swisstopo.geologie-geowege'
+    id = Column('post_id', Integer, primary_key=True)
```
Comment (ltclm) [inline chsdi/models/vector/stopo.py:356]: you could change the post_id index in the database to a unique index:
```sql
CREATE UNIQUE INDEX idx_erlebnisgeologie_post_id
    ON geol.erlebnisgeologie USING btree
    (post_id ASC NULLS LAST)
    TABLESPACE pg_default;
```

**Pull request merged**